src/charm.py

```diff
@@ -144,7 +143,7 @@ def __init__(self, *args):
             relation_name="alertmanager",
         )
 
-        external_url = urlparse(self.external_url)
+        urlparse(self.external_url)
```

Comment:
this line is odd, we should add a comment to explain what's going on

Reply:
Removed in next commit.